### PR TITLE
Update README header

### PR DESCRIPTION
**Pull request by kaishin**
- Increase logo mark size
- Add license and version badges

### CODE REVIEW

**Comment by robrix** [comment]
What’s the motivation for increasing the size of the logo? It ends up pushing the content of the readme down quite a bit further.

**Comment by neilpa** [comment]
Yea, this creates a lot of dead space and looks odd to me.

**Comment by neilpa** [comment]
The license and version badges are nice though.

**Comment by kaishin** [comment]
> What’s the motivation for increasing the size of the logo? It ends up pushing the content of the readme down quite a bit further.

Perviously neither the mark not the title had room to breath and their horizontal alignment was awkward. Pushing the content down further would have been a regression if the content was visible to begin with, which is not the case. In general, vertical spacing is never an issue for users on the Web, as long as they are aware that there is more content below.

> Yea, this creates a lot of dead space and looks odd to me.

You say dead space, I say white space ;)

I can make the icon smaller but we can't make it as small as it used to be for clarity.

**Comment by neilpa** [comment]
> You say dead space, I say white space ;)

Probably the Tetris part of my brain kicking in

**Comment by jspahrsummers** [comment]
Would the logo fit in better after the heading?

**Comment by kaishin** [comment]
I am actually thinking of trying a full-width image with the logo centered. I've seen this work better in a few repositories like https://github.com/Alamofire/Alamofire.

**Comment by jspahrsummers** [comment]
@kaishin That would be cool. :sunglasses:

**Comment by kaishin** [comment]
Updated with a full-width header. I abstained from adding a logotype because I didn't want to commit to a typeface. Preview below.

screenshot-2015-06-22

**Comment by robrix** [comment]
Wow! :heart_eyes:

**Comment by neilpa** [comment]
That's awesome

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# <img src="https://cloud.githubusercontent.com/assets/432536/5252404/443d64f4-7952-11e4-9d26-fc5cc664cb61.png" width="36" height="36"> Carthage
+<img src="https://raw.githubusercontent.com/carthage/carthage/master/Logo/PNG/header.png">
```
Comment (jspahrsummers) [inline README.md]: Can you change this to ``, which will work per-branch?

**Comment by jspahrsummers** [comment]
@kaishin That looks amazing, thanks! :sparkles:

File: README.md (hunk 1 above)
Comment (kaishin) [inline README.md]: Whoa, didn't know this would work. That's exactly what I wanted.

**Comment by kaishin** [comment]
@jspahrsummers Done. I also added license info for the backdrop picture, let me know if you are cool with that.

**Comment by robrix** [comment]
I think the license is _entirely_ appropriate :bow:

**Comment by jspahrsummers** [comment]
That license is philosophically fine with me, but I'm concerned that it contradicts the overall license for the project (which permits commercial, non-share-alike use).

That said, we can just note it in LICENSE.md as well, and I think it'd be fine.

**Comment by kaishin** [comment]
@jspahrsummers I thought about that as well, but it's not uncommon to have graphic assets excluded from the main license, especially if they are building on top of other assets. Logos and trademarks are typically not permitted for commercial use (as in, selling the Carthage logo as is, etc). WebKit and Mozilla are prime examples.

I'll add that to LICENSE.

**Comment by jspahrsummers** [comment]
:sparkles:

**Pull request merged**